freyja_apps/template/Cargo.template.toml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+[package]
+name = "freyja-template-app"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+
+[dependencies]
+# These two dependencies are required for anyone implementing a Freyja application
```

Comment:
Similar to my other comment: consider adding a comment about how to choose the commit hash here

Reply:
Ah, now I see it's in the documentation. Up to you if you want to add it here too

Reply:
I'll leave it to the docs